### PR TITLE
chore (travis) Update travis ci to run tests

### PR DESCRIPTION
**Pull request by ruganda**
#### What does this PR do?
- Update travis ci to run tests
#### Description of Task to be completed?
- update travis ci to run tests
- add a Travis ci readme badge
#### How should this be manually tested?
- NA
#### What are the relevant pivotal tracker stories?
[160814315](https://www.pivotaltracker.com/story/show/160814315)

### CODE REVIEW

**Comment by cmplx-xyttmt** [comment]
@ruganda, good work.
A small issue is that in the readme, the badge is an svg instead of markdown, I don't think it will render correctly.

**Comment by marthamareal** [comment]
> @ruganda, good work.
> A small issue is that in the readme, the badge is an svg instead of markdown, I don't think it will render correctly.

Also @ruganda that seem to be a link to the badge, maybe you can consider adding the real travis badge on the readme.

**Comment by ruganda** [comment]
@cmplx-xyttmt , @marthamareal , Thanks. let me change the badge to a markdown

**Comment by ruganda** [comment]
I have replaced the svg with a markdown

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - stable
+
+install:
+  - npm install
+
+script:
+  - npm test
+
+after_success:
+  - npm run coveralls
```
Comment (inno-asiimwe) [inline .travis.yml]: What does this do?

**Comment by ruganda** [comment]
Hello @inno-asiimwe , This `PR` is also ready to be merged. The build is `passing` and I need it to be merged so that I can work on `coverage`

**Pull request merged**